core/controllers/question_test.py

```diff
@@ -32,6 +32,41 @@ class QuestionsHandlersTest(test_utils.GenericTestBase):
     method of question manager handler.
     """
 
+    def _create_valid_question_data(self, default_dest_state_name):
+        """Creates a valid question_data dict.
+
+        Args:
+            default_dest_state_name: str. The default destination state.
+
+        Returns:
+            dict. The default question_data dict.
+        """
+        state = exp_domain.State.create_default_state(
+            default_dest_state_name).to_dict()
```

Comment:
Better to create a valid domain object and then do a single to_dict() at the end. That gives you more validation/guarantees.

(Generally, try to work with domain objects, not dicts.)

Reply:
Done.